assets/xml/objects/object_tab.xml

```diff
@@ -1,63 +1,63 @@
-﻿<Root>
+<Root>
     <File Name="object_tab" Segment="6">
```

Comment:
Can you put a comment above the `<File>` node that explains what assets are in this file? There are a lot of examples in the repo if you need an example of this, but feel free to ask if you need a specific example!